contract-addresses/mainnetAddresses.json

```diff
@@ -3,10 +3,15 @@
     "aaveAddress" : {"artifact" : "IERC20", "address" : "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9"},
     "aaveEthPCVDepositAddress": { "artifact": "AavePCVDeposit", "address":  "0x5B86887e171bAE0C2C826e87E34Df8D558C079B9"},
     "aaveEthPCVDripControllerAddress": { "artifact": "PCVDripController", "address":  "0xb3D63876d95d3a5e591D4DE536dC410b97244086"},
+    "aaveGovernanceV2Address" : {"artifact" : "IAaveGovernanceV2", "address" : "0xEC568fffba86c094cf06b22134B23074DFE2252c"},
     "aaveIncentivesControllerAddress" : {"address": "0xd784927ff2f95ba542bfc824c8a8a98f3495f6b5"},
-    "aaveLendingPoolAddress" : {"address": "0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9"},
+    "aaveLendingPoolAddress" : {"artifact" : "ILendingPool", "address": "0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9"},
     "aaveRaiPCVDeposit": {"artifact" : "AavePCVDeposit", "address": "0xd2174d78637a40448112aa6B30F9B19e6CF9d1F9"},
-    "aaveTribeIncentivesControllerAddress" : {"address": "0xFF865335401F12B88fa3FF5A3a51685A7f224191"},
+    "aaveTribeIncentivesControllerAddress" : {"artifact": "IAaveIncentivesController", "address": "0xDee5c1662bBfF8f80f7c572D8091BF251b3B0dAB"},
+    "aaveTribeIncentivesControllerProxyAddress" : {"artifact": "TransparentUpgradeableProxy", "address": "0xDee5c1662bBfF8f80f7c572D8091BF251b3B0dAB"},
```

Comment:
is it intentional that proxy address is same as non-proxy address

Reply:
This is sort of a weird nuance. They are the same, one is just using the abi for the Proxy and the other is using the abi for the logic contract